App/Author/author.rb

```diff
@@ -0,0 +1,46 @@
+require 'json'
+
+class Author
+  attr_accessor :id, :firstname, :lastname, :items
+
+  def initialize(firstname, lastname)
+    @firstname = firstname
+    @lastname = lastname
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end

```

Comment:
- [x] @Team great work adding the item method to cover the 1- many relationship, however you forgot to add self , as part of what is stored in the item like you did for the genre and label classes eg  
```suggestion
   def add_item(item)
    @items << item
    item.author = self
  end

```

Reply:
finished! ✔️